disasm/disasm.cc

```diff
@@ -1412,6 +1412,12 @@ void disassembler_t::add_instructions(const isa_parser_t* isa)
     DISASM_INSN("c.fsw", c_fsw, 0, {&rvc_fp_rs2s, &rvc_lw_address});
     DISASM_INSN("c.fswsp", c_fswsp, 0, {&rvc_fp_rs2, &rvc_swsp_address});
   }
+  else if (isa->extension_enabled(EXT_ZCMLSD)) {
```

Comment:
move the `else` onto the same line as the previous closing brace

Reply:
fixed in 9b27b67